player/js/renderers/CanvasRendererBase.js

```diff
@@ -273,7 +273,7 @@ CanvasRendererBase.prototype.destroy = function () {
   var i;
   var len = this.layers ? this.layers.length : 0;
   for (i = len - 1; i >= 0; i -= 1) {
-    if (this.elements[i]) {
+    if (this.elements[i] && this.elements[i].destroy()) {
```

Comment:
I think this should be `this.elements[i] && this.elements[i].destroy` instead, as in the other code change

Reply:
Yes, you are right. I fix this